skyvern/schemas/runs.py

```
@@ -416,6 +420,12 @@ def validate_urls(cls, url: str | None) -> str | None:
             return None
         return validate_url(url)
 
+    @model_validator(mode="after")
+    def validate_browser_reference(cls, values: WorkflowRunRequest) -> WorkflowRunRequest:
+        if values.browser_session_id and values.browser_profile_id:
+            raise ValueError("Cannot specify both browser_session_id and browser_profile_id")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)